### DISCUSSION TITLE
Issues connecting with Java client

### DISCUSSION THREAD

**Discussion by dj5483** [piotr-rojek/devopsifyme-sbemulator#19] [General]
I've got the emulator running locally with rabbitmq - running a java client as a sender using the azure-java-sdk.  Having an issue in the emulator 

`close(error:error(condition:amqp:internal-error,description:Unable to cast object of type 'System.UInt64' to type 'System.String'.))`

the java sdk does not have the EnableAmqpLinkRedirect option, so can not set that to false per the docuementation.

Has anyone experienced this? is the LinkRedirect option a potential culprit?

On a side note, was looking to try the local backend vs. Rabbit - is there documentation for enabling local only?  been browsing source, but haven't figured that out yet.

**Comment by ederaildo** [comment]
Same error :(

**Comment by julian9191** [comment]
Has anyone been able to find a workaround for this?
